lib/amazon-pricing/helpers/instance-type.rb

```diff
@@ -155,7 +157,17 @@ def family_members(api_name)
 
 
       def api_name_to_nf(name)
-        size_to_nf[name.split('.').last]
+        type = name.split('.').last
+        if (type == METAL)
+          # try to get largest size supported for family: presumes METAL is *not* in size_to_nf hash
+          # assumes family_members are sorted by size
+          sizes = family_members(name)
```

Comment:
is family_members sorted by size?

Reply:
yes:  @@INSTANCE_TYPES_BY_CLASSIFICATION arrays of sizes, are currently sorted by size, and is what is returned. (I also added a comment there saying these arrays need to be kept in size sorted order.)